bindata/v4.1.0/config/defaultconfig.yaml

```diff
@@ -15,6 +15,10 @@ extendedArguments:
   - "true"
   flex-volume-plugin-dir:
   - "/etc/kubernetes/kubelet-plugins/volume/exec" # created by machine-config-operator, owned by storage team/hekumar@redhat.com
+  pv-recycler-pod-template-filepath-nfs: # avoid using default template, even though recycler reclaim policy is deprecated
+  - "/etc/kubernetes/manifests/recycler-pod.yaml"
```

Comment:
Where this file comes from? I assume you guys are creating it, can you make a comment similar to the one above?

Reply:
Sure, added comment to state it's created by MCO